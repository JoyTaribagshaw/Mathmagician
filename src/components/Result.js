import React from 'react';
import PropTypes from 'prop-types';

function Result({ value }) {
  return <div className="input">{value}</div>;
}

Result.propTypes = {
  value: PropTypes.string.isRequired,
};

export default Result;
