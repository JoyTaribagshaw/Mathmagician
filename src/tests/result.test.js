import React from 'react';
import { render } from '@testing-library/react';
import Result from '../components/Result';

describe('Result Component', () => {
  test('renders correctly', () => {
    const { container } = render(<Result value="0" />);
    expect(container).toMatchSnapshot();
  });
});
