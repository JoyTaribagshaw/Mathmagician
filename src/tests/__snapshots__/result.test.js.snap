// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Component renders correctly 1`] = `
<div>
  <div
    class="input"
  >
    0
  </div>
</div>
`;
